### DISCUSSION TITLE
How to hide Jupyter notebook language in Top language card?

### DISCUSSION THREAD

**Discussion by adimail** [anuraghazra/github-readme-stats#3679] [Q&A] [answered]
I have a single repo with jupyter notebook, but the notebooks takes too much capacity. How do I hide it from the card? I tried `&hide=jupyter-notebook`,  `&hide=jupyternotebook`, `&hide=JupyterNotebook`, `&hide=jupyter`

But nothing works. Please help me here. Thanks

**Comment by qwerty541** [comment, marked as answer]
Hey, @adimail! You can hide jupyter notebook using `&hide=jupyter%20notebook`. Query string value should be URL encoded. I will add an example below.

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs?username=adimail&hide=jupyter%20notebook" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs?username=adimail&hide=jupyter%20notebook" />

**Reply by adimail** [reply]
Thanks a lot

**Reply by qwerty541** [reply]
Glad to help! Feel free to contact me if you need any further assistance.

**Reply by RYANFRANKLIN237** [reply]
thanks for your help, i have been looking for a solution for days
